### PR TITLE
Feature/form styles

### PR DESCRIPTION
**Pull request by itsnameissteven**
## Is this a fix or a feature?
Feature

## What is the change?
* Initial SCSS file structure was set up
  * All new SCSS files should be imported to index.scss
* Styles have been added to the form 
  * We can lighten up some css(animations) with some refactoring in the form component.
* Progress bar added to the registration page. 
* Node SASS added as dependency. 
* Input type was added for fields
## What does it fix?
n/a
## Where should the reviewer start?
Review how SCSS files are organized and imported.
Review Form.scss and _base.scss as these are the biggest changes
## How should this be tested?
fetch locally, run npm i, open in the browser(in mobile view) fill out the registration form.
## Relevant ScreenShots

Screen Shot 2021-05-13 at 10 26 17 PM

### CODE REVIEW

File: src/components/Form/Form.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 5px;
+  background: $lightAccent;
+  transform-origin: left;
+  animation-duration: 0.3s;
+  animation-timing-function: ease-out;
+}
```
Comment (alexthompson207) [inline src/components/Form/Form.scss:42]: This is some SCSS magic, great work!

File: src/components/Form/Form.tsx [hunk 2]
```diff
@@ -31,14 +32,24 @@ const Form: React.FC<FormProps> = ({ header, page, setPage, inputs }): JSX.Eleme
...
-    <form onSubmit={e => handleSubmit(e)}>
-      <h1>{header}</h1>
-      {allInputs}
-      <button
-        // onClick={() => setPage(page + 1)}
-        type='submit'
-      >Next</button>
+    <form onSubmit={e => handleSubmit(e)} className="register-form">
+      <h1 className="register-form__header">{header}</h1>
+      <div className="register-form__inputs">{allInputs}</div>
```
Comment (alexthompson207) [inline src/components/Form/Form.tsx:45]: Great job adding classes dynamically!

**Review by alexthompson207 — APPROVED** [review]

File: src/components/Form/Form.scss (hunk 1 above)
Comment (domo2192) [reply src/components/Form/Form.scss:42]: What the hell is going on here..Looks great!!!

**Review by domo2192 — APPROVED** [review]
Really cool stuff..no clue whats happening

File: src/components/Form/Form.scss (hunk 1 above)
Comment (PaigeVannelli) [reply src/components/Form/Form.scss:42]: Looks great!

File: src/components/Form/Form.tsx (hunk 2 above)
Comment (PaigeVannelli) [reply src/components/Form/Form.tsx:45]: Nice job with the classnames!

**Review by PaigeVannelli — APPROVED** [review]

**Pull request merged**